### PR TITLE
Update RSU counts query to the new table schema

### PR DESCRIPTION
**Pull request by drewjj**
Includes the updates to the cloud function for rsu counts so that the new table schema can be supported.

Still grabs yesterday's counts (UTC)

### CODE REVIEW

**Comment by drewjj** [comment]
One other thing to note, this is hard coded for BSM counts only. This was made to be able to support other message types in the future when we want to.

File: GCP_cloud_functions/rsu-manager/rsu-querycounts/main.py [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+            f"GROUP BY RSU, Road " \
             f"ORDER BY Road, RSU"
     query_job = client.query(query)
 
     result = {}
     for row in query_job:
       item = {}
       item["road"] = row["Road"]
-      item["count"] = row["BsmCount"]
+      item["count"] = row["Count"]
```
Comment (payneBrandon) [inline GCP_cloud_functions/rsu-manager/rsu-querycounts/main.py:26]: What do you think about adding in the 'yesterday' value here as a 'Date' field? That way we know for sure what date is being applied when we hit the service?

Comment (drewjj) [reply GCP_cloud_functions/rsu-manager/rsu-querycounts/main.py:26]: I added this to the function. I can add this to the RSU Manager to display too but maybe we should wait to add it to the RSU Manager so we can decide where the most useful place to put this value might be.

Comment (payneBrandon) [reply GCP_cloud_functions/rsu-manager/rsu-querycounts/main.py:26]: Sounds reasonable to me!

**Review by payneBrandon — APPROVED** [review]

**Pull request merged**